### PR TITLE
My first page

### PR DESCRIPTION
**Pull request by adelina2999**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,22 +1,105 @@
...
-      <li>Luni</li>
-      <li>Marti</li>
-      <li>Miercuri</li>
-      <li>Joi</li>
-      <li>Vineri</li>
-      <li>Sambata</li>
-      <li>Duminica</li>
-    </ol>
+  <body background="bg-image.jpg">
+      <h1 class="intro">Welcome! I am a Reign fan, exactly like you!</h1>
```
Comment (vadim11041996) [inline index.html]: you must specify the path like that
`Images/bg-image.jpg` 

your  index.html file is at the same level as Image, but image you're trying to acces is inside of `Images` folder

File: index.html [hunk 2]
```diff
@@ -1,22 +1,105 @@
...
+            <th>Caracter Name</th>
+            <th>Real Name</th>
+            <th>Episode Number</th>
+            <th>Photo</th>
+          </tr>
+          <tr>
+            <td>Mary Stuart</td>
+            <td>Adelaide Kane</td>
+            <td>78 episodes</td>
+            <td><img src="adelaide.jpg" alt="Adelaide Kane"></td>
```
Comment (vadim11041996) [inline index.html]: same here with image, use the full path name as I wrote above

File: script.js [hunk 3]
```diff
@@ -1 +1 @@
-console.log('From javascript ')
+alert("This is a page created for the Reign fans! Enjoy!");
```
Comment (vadim11041996) [inline script.js]: all the files must have in the end a new empty line, add please a new empty line

File: index.html (hunk 1 above)
Comment (adelina2999) [reply index.html]: It works. Thank you very much.

File: script.js (hunk 3 above)
Comment (adelina2999) [reply script.js]: What?

File: script.js [hunk 4]
```diff
@@ -1 +1,5 @@
-console.log('From javascript ')
+function myFunction() {
+    alert("Hello World!");
+  }
+  myFunction()
+  ;
```
Comment (vadim11041996) [inline script.js]: check for error, wrong place for `;`

File: style.css [hunk 5]
```diff
@@ -1,11 +1,29 @@
...
+  width: 70px;
+}
+
+.th {
+  background-color: #4CAF50;
+  color: white;
+  text-align: center;
+}
+
+
```
Comment (vadim11041996) [inline style.css]: don't forget, all the files must have at the end a new empty line

File: script.js (hunk 3 above)
Comment (adelina2999) [reply script.js]: Why?

File: script.js (hunk 4 above)
Comment (adelina2999) [reply script.js]: Done

File: script.js (hunk 3 above)
Comment (vadim11041996) [reply script.js]: image
github is complaining

**Pull request merged**